docs/sources/explore/index.md

```diff
@@ -300,9 +302,15 @@ Clicking anywhere on the span row shows span details.
 - Process metadata: Metadata about the process that logged this span.
 - Logs: List of logs logged by this span and associated key values. In case of Zipkin logs section shows Zipkin annotations.
 
-## Navigating between Explore and a dashboard
+##### Trace to logs
 
-To help accelerate workflows that involve regularly switching from Explore to a dashboard and vice-versa, we've added the ability to return to the origin dashboard
+> This feature is only available in Grafana 7.4+.
+
+Trace to logs allows you to navigate from a trace span to the selected data source's log.
```

Comment:
Would expand this a little bit. Something like:

"You can navigate from a span in a trace view directly to logs relevant for that span. This is available for Tempo, Jaeger and Zipkin data source at this moment. See their relevant documentation for instruction how to configure this feature."

<Image of the icon in the trace view>

"Click the document icon to open a split view in Explore with the configured data source and query relevant logs for the span"